electron/controller.js

```diff
@@ -118,43 +101,23 @@ class ZapController {
     }
   }
 
-  killAllZombieProcesses() {
+  /**
+   * Terminate any processes known to have been started by the app.
+   */
+  killAllSpawnedProcesses() {
     Object.keys(this.processes).forEach(key => {
       const pid = this.processes[key]
       if (pid) {
-        mainLog.debug(`Killing zombie ${key} process with pid ${pid}`)
+        mainLog.debug(`Killing ${key} process with pid ${pid}`)
         try {
           process.kill(pid)
         } catch (e) {
-          mainLog.warn(`Unable to kill zombie ${key} process with pid ${pid}`)
+          mainLog.warn(`Unable to kill ${key} process with pid ${pid}`)
         }
         this.processes[key] = null
```

Comment:
A minor style suggestion. Consider placing this under finally so `try` feels like a monolithic structure

Reply:
Updated.

I also fixed the `ipcMain.on('processExit'` handler as per slack discussion.